app/models/user.rb

```diff
@@ -1,7 +1,8 @@
 class User < ActiveRecord::Base
   has_many :wikis
   has_many :collaborations
-  has_many :private_wikis, through: :collaborations
+  has_many :private_wikis, through: :collaborations, source: :wiki
+  
```

Comment:
Quick remark: `private_wikis` may not be that good of a name, because a user could totally be the *owner* of one or several private wikis, and one could expect `some_user.private_wikis` to return that list of owned, private wikis, instead of a list of invited-to, private wikis.

One could simply rename to something more explicit:

```ruby
has_many :wiki_invites, through: :collaborations, source: :wiki
```